src/docker/docker-compose.yml

```diff
@@ -5,6 +5,7 @@ services:
   # - TEST_POSTGRES_URI=postgres://prisma:prisma@postgres/tests
   # - TEST_POSTGRES_ISOLATED_URI=postgres://prisma:prisma@postgres_isolated:5435/tests
   # - TEST_MYSQL_URI=mysql://prisma:prisma@mysql:3306/tests
+  # - TEST_MYSQL_ISOLATED_URI=mysql://prisma:prisma@mysql_isolated:3306/tests
```

Comment:
this should be removed @williamluke4 this var is not needed there

Reply:
Aren't those just comments

Reply:
Ok I see that the isolated mysql was also added before ✅